OpenOversight/app/templates/tutorial.html

```diff
@@ -0,0 +1,40 @@
+{% extends "base.html" %}
+
+{% block content %}
+
+<div class="container theme-showcase" role="main">
+    <h1>OpenOversight Labeling Tutorial</h1>
+
+        <p>We want you to help us go through images submitted by activists
+        and Lucy Parsons Labs' social media monitoring of law enforcement
+        to find faces of police officers and match them to officers in our
+        roster.</p>
+
+        <h2>Step 1: Find Police Officers</h2>
+        <p>When an image is first displayed, see if you see any faces
+          of police officers. If you do, see if you can make out anything
+          about their badge number or name. If you can even see a few letters or
+          numbers, proceed to Step 2 by clicking "Police officers are clearly
+          visible with badge numbers or names".</p>
+        <p>If you don't see any clear faces of police officers in the image
+          with names or badge numbers, just click
+          "No police officers with badge numbers or names" to advance to the next
+          image. However, if you do then you will proceed to Step 2.
+          </p>
+
+        <h2>Step 2: Map Police Faces to Entries in the Roster</h2>
+        <p>We want you to use the cropper to first select an officer face in the
```

Comment:
I rewrote the tutorial and added screenshots in commit c96ea74aaa37829d3ecbf6445d25ab54a78ab8cf (the word cropper no longer appears 😜 ).